test/avocado/timeoutlib.py

```diff
@@ -114,7 +117,7 @@ def __wrap(*args, **kwargs):
                         if not self.inverse:
                             return output
 
-                    except self.exceptions as e:
+                    except (self.exceptions + (TimeoutError,)) as e:
```

Comment:
here, for example.

Reply:
Fixed all of these to be without comma now.